bitnami/minio/README.md

```diff
@@ -118,6 +118,7 @@ The following table lists the configurable parameters of the MinIO&reg; chart an
 | `securityContext.enabled`         | Enable security context                                                                   | `true`                         |
 | `securityContext.fsGroup`         | Group ID for the container                                                                | `1001`                         |
 | `securityContext.runAsUser`       | User ID for the container                                                                 | `1001`                         |
+| `securityContext.runAsNonRoot`    | Avoid running as root User                                                                | `false`                        |
```

Comment:
Running containers as non root is a best practices in terms of securizing containers, so I think the default approach should be the opposite, by default the security context should be set to use `true` as value for `runAsNonRoot`
```suggestion
| `securityContext.runAsNonRoot`    | Run container as non-root user                                                        | `true`                         |
```

+info: https://docs.bitnami.com/tutorials/work-with-non-root-containers/